### PR TITLE
meme: 2017-09-10 -> 2020-05-28

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Can you mention in the commit that you changed it to buildGoModule?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104398` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme</li>
  </ul>
</details>

**Comment by fgaz** [comment]
@SuperSandro2000 done

File: pkgs/applications/graphics/meme/default.nix [hunk 1]
```diff
@@ -1,19 +1,21 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv
+, buildGoModule
+, fetchFromGitHub
+}:
 
-buildGoPackage rec {
+buildGoModule {
   pname = "meme-unstable";
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/meme/default.nix:7]:
```suggestion
  pname = "meme";
```

File: pkgs/applications/graphics/meme/default.nix [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-  version = "2017-09-10";
-
-  owner = "nomad-software";
-  repo = "meme";
-  goPackagePath = "github.com/${owner}/${repo}";
+  version = "2020-05-28";
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/meme/default.nix:8]:
```suggestion
  version = "unstable-2020-05-28";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/graphics/meme/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/applications/graphics/meme/default.nix]: It was fixed in master, I rebased.

File: pkgs/applications/graphics/meme/default.nix (hunk 2 above)
Comment (fgaz) [reply pkgs/applications/graphics/meme/default.nix]: Same

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104398` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104398` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme</li>
  </ul>
</details>

**Pull request merged**